docs/source/tutorial.md

```diff
@@ -0,0 +1,49 @@
+# Tutorial
+
+This page covers brief tutorials on how to use pycytominer-transform.
+
+## Data Sources
+
+```{eval-rst}
+Data sources may be provided to pycytominer-transform using local filepaths or remote object-storage filepaths (for example, AWS S3, GCP Cloud Storage, Azure Storage).
+We use `cloudpathlib <https://cloudpathlib.drivendata.org/~latest/>`_  under the hood to reference files in a unified way, whether they're local or remote.
+
+Remote object storage paths which require authentication or other specialized configuration may use cloudpathlib client arguments (`S3Client <https://cloudpathlib.drivendata.org/~latest/api-reference/s3client/>`_, `AzureBlobClient <https://cloudpathlib.drivendata.org/~latest/api-reference/azblobclient/>`_, `GSClient <https://cloudpathlib.drivendata.org/~latest/api-reference/gsclient/>`_) and :code:`convert(..., **kwargs)` (:mod:`convert() <pycytominer_transform.convert.convert>`).
+For example, remote AWS S3 paths which are public-facing and do not require authentication (like, or similar to, :code:`aws s3 ... --no-sign-request`) may be used via :code:`convert(..., no_sign_request=True)` (:mod:`convert() <pycytominer_transform.convert.convert>`).
+```
+
+## Data Conversion Types
+
+- __Apache Parquet__: "Apache Parquet is an open source, column-oriented data file format designed for efficient data storage and retrieval. It provides efficient data compression and encoding schemes with enhanced performance to handle complex data in bulk." ([reference](https://parquet.apache.org/))
+
+## CellProfiler CSV Output to Parquet
+
+CellProfiler pipelines or projects may produce various CSV-based component output (for example, "Cells.csv", "Cytoplasm.csv", etc.).
+Pycytominer-transform converts this data to Parquet from local or object-storage based locations.
+
+Files with similar names nested within sub-folders will be concatenated by default (appended to the end of each data file) together and used to create a single Parquet file per data target.
+For example: if we have `folder/subfolder_a/cells.csv` and `folder/subfolder_b/cells.csv`, using `convert(source_path="folder", ...)` will result in `folder.cells.parquet` (unless `concat=False`).
+
+For example, see below:
+
+```python
+from pycytominer_transform import convert
+
+# using an local path
+convert(
+    source_path="./local/file/path",
+    source_datatype="csv",
+    dest_path=".",
+    dest_datatype="parquet",
+)
+
+# using an s3-compatible path with no signature for client
+convert(
+    source_path="s3://s3path",
+    source_datatype="csv",
+    dest_path=".",
```

Comment:
Dest_path at the moment is restricted to local-only paths. You can mix cloud sources and local destinations. We should outline this within the documentation to be more explicit here, I'll add this into a forthcoming change.

Enabling remote sources and destinations alongside remote workers (for instance, within a Dask or Ray cluster) would help scale the use of this tool. I'm envisioning scenarios where one would have the option to use this tool as an actor but not rely on where execution was initiated for local resources (besides network access). Updating this functionality for dest_path cloud locations I feel could be a separate PR due to how this might operate, open to your thoughts here.

As a sketch of the above statements, consider the following "layered" workflow (as a future state of pycytominer-transform). Note that storage sources and destinations wouldn't necessarily need to be the same.

```mermaid
flowchart LR
subgraph actor["Actor / Initiator"]
pycytominer-transform
end

subgraph Storage
direction TB
subgraph sourcestorage["Source"]
s3source[S3 or Cloud Source]
end
subgraph deststorage["Destination"]
s3dest[S3 or Cloud Destination]
end
end
subgraph worker["Worker"]
dask[Dask or Ray Cluster]
end
pycytominer-transform --> |initiates work| dask
dask --> |creates data| s3dest
s3source --> |gathers data| dask


```

Reply:
Referencing this diagram and potential future state for documentation needs within #17